dali/dfu/dfuserver.cpp

```diff
@@ -222,24 +241,59 @@ int main(int argc, const char *argv[])
             if (!*q)
                 break;
         }
-        q = globals->queryProp("@MONITORQUEUE");
-        if (q&&*q) {
+        globals->getProp("@monitorqueue", monitorqueue);
+#else
+        getDfuQueueName(queue, name);
+        unsigned maxJobs = globals->getPropInt("@maxJobs", 1);
+        if (maxJobs<1)
+        {
+            OERRLOG("maxJobs is %u", maxJobs);
+            return 1;                       
+        }
+        VStringBuffer mask("Queue[@name=\"%s\"][1]", queue.str());
+        if (!stop)
+        {
+            if (serverstatus->queryProperties()->queryPropTree(mask.str()))
+            {
+                OERRLOG("Queue already active: %s", queue.str());
+                return 1;
+            }
+            else
+            {
+                IPropertyTree *t = createPTree();
+                t->setProp("@name",queue);
+                t->setPropInt("@num", maxJobs);
+                serverstatus->queryProperties()->addPropTree("Queue",t);
+            }
+            serverstatus->commitProperties();
+            engine->setDefaultTransferBufferSize((size32_t)globals->getPropInt("@transferBufferSize"));
+            for (unsigned i=0; i<maxJobs; i++)
+                engine->startListener(queue,serverstatus);
+        }
+        else
+            stopDFUserver(queue);
+        getDfuMonitorQueueName(monitorqueue, name);
+#endif
+        if (monitorqueue.length()>0)
+        {
             if (stop) {
-                stopDFUserver(q);
+                stopDFUserver(monitorqueue);
             }
             else {
                 IPropertyTree *t=serverstatus->queryProperties()->addPropTree("MonitorQueue",createPTree());
-                t->setProp("@name",q);
-                engine->startMonitor(q,serverstatus,globals->getPropInt("@MONITORINTERVAL",60)*1000);
+                t->setProp("@name",monitorqueue);
+                int monitorInterval = globals->getPropInt("@monitorinterval", 60);
```

Comment:
The dfuserver.xsd and dfuserver.xsl have the lowercased monitorinterval and monitorqueue.  I didn't change it earlier because I'd have to change the xsd too.  I will change if change doesn't affect backwards compatibility.

Reply:
I have confirmed that the camel casing these fields affect backward compatibility. I am having to leave the environment.xml unchanged and leaving the .xsd with the non-camel case version.  I'm modifying the  .xsl to generate the camel cased fields from the non-camel cased .xml.